source/getting_started.rst

```diff
@@ -122,6 +126,9 @@ In the MATLAB Command Window type::
 This should run an example case using the Reference Model 3 (RM3) point absorber. 
 A SimMechanics Explorer window will open within the MATLAB window, and figures will be generated displaying simulation outputs. 
 
+.. Note:: 
+	Adam:
+	 Change the test instructions to just running wsTest.m (also this file should be renamed to wecSimTest.m for naming consistency)
 
```

Comment:
This is executed by the MATLAB builtin `runtests` function. I just added a Software Tests section with PR #434, but feel free to add more detail. Also, I'm open to changing the file name for consistency.